bin/shpec

```diff
@@ -1,41 +1,51 @@
-#!/usr/bin/env bash
-VERSION=0.1.2
-examples=0
-failures=0
-test_indent=0
-red="\033[0;31m"
-green="\033[0;32m"
-norm="\033[0m"
+#
+: Use one of: bash, dash, or ksh93
```

Comment:
I don't know what I need to do to zsh to make it accept pure POSIX shell.
It's a powerful shell for interactive use, and has some nifty features, but I gave up on zsh long ago for scripting.
Sorry.